subject_behavdata.py

```diff
@@ -2,273 +2,414 @@
-    # -1 = control task; 0 = missed trial; 1 = wrong source (image recognized but wrong quadrant remembered);
-    #2 = image recognized with correct source
+    # calculate post-scan source (position) accuracy;
+    #  -1 = control task; 0 = missed trial; 1 = wrong source (image recognized but wrong quadrant remembered);
+    # 2 = image recognized with correct source
     mainEnc['position_accuracy'] = 0
-    for j in mainEnc[mainEnc['recognition_accuracy']==1].index:
+    for j in mainEnc[mainEnc['recognition_accuracy'] == 1].index:
         if mainEnc.loc[j, 'position_correct'] == mainEnc.loc[j, 'position_response']:
             mainEnc.loc[j, 'position_accuracy'] = 2
         else:
             mainEnc.loc[j, 'position_accuracy'] = 1
-    #import source accuracy info from mainEnc into ret (in-place)
-    for i in ret[ret['old_new']=='OLD'].index:
+    # import source accuracy info from mainEnc into ret (in-place)
+    for i in ret[ret['old_new'] == 'OLD'].index:
         picID = ret.loc[i, 'stim_id']
         ret.loc[i, 'position_correct'] = mainEnc.loc[picID, 'position_correct']
-        ret.loc[i, 'position_accuracy'] = mainEnc.loc[picID, 'position_accuracy']
-    #reset mainEnc searchable index to default
-    #and re-order columns to match order in mainCTL
+        ret.loc[i, 'position_accuracy'] = mainEnc.loc[picID,
+                                                      'position_accuracy']
+    # reset mainEnc searchable index to default
+    # and re-order columns to match order in mainCTL
     mainEnc.reset_index(level=None, drop=False, inplace=True)
-    cols = ['trial_number', 'onset', 'duration', 'offset', 'trial_type', 'response',
-       'response_time', 'stim_id', 'stim_file', 'stim_category',
-       'stim_name', 'recognition_accuracy', 'recognition_responsetime',
-       'position_correct', 'position_response', 'position_accuracy', 'position_responsetime']
+    cols = ['trial_number', 'onset', 'duration', 'offset', 'trial_type',
+            'response', 'response_time', 'stim_id', 'stim_file',
+            'stim_category', 'stim_name', 'recognition_accuracy',
+            'recognition_responsetime', 'position_correct',
+            'position_response', 'position_accuracy', 'position_responsetime']
     mainEnc = mainEnc[cols]
-    #Re-merge mainEnc and mainCTL and re-order by trial number
+    # Re-merge mainEnc and mainCTL and re-order by trial number
     mainMerged = mainEnc.append(mainCTL, ignore_index=True)
-    mainMerged.sort_values('trial_number', axis=0, ascending=True, inplace=True)
+    mainMerged.sort_values('trial_number', axis=0, ascending=True,
+                           inplace=True)
     return mainMerged
 
+
 def extract_taskFile(bID, sID, file_list, output):
-    #import data from three text files into pandas DataFrames
+    """
+    Parameters:
+    ----------
+    bID: string (behavioral ID)
+    sID: string (structural ID)
+    file_list: list (three input files)
+    output: string (output Folder)
+
```

Comment:
For added lines 343-344:
343: "bID: string (subject PSCID, id used during data collection)"
344: "sID: string (subject DCCID, id used in Loris)"